jupyter/rocm/pytorch/ubi9-python-3.11/kustomize/base/kustomization.yaml

```diff
@@ -1,9 +1,9 @@
 ---
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
-namePrefix: rocm-jupyter-pytorch-ubi9-python-3-11-
+namePrefix: jupyter-rocm-pytorch-ubi9-python-3-11-
 commonLabels:
-  app: rocm-jupyter-pytorch-ubi9-python-3-11
+  app: jupyter-rocm-pytorch-ubi9-python-3-11
```

Comment:
Hey, didn't check - what does this affects? Is it safe? :thinking:

Reply:
does not run without this change in gha, and I did not manage to make the tests run in openshift-ci, so, yeah, I think it makes things better overall